src/CMakeLists.txt

```diff
@@ -578,6 +603,9 @@ if(BUILD_SERVER)
 	if (USE_SPATIAL)
 		target_link_libraries(${PROJECT_NAME}server ${SPATIAL_LIBRARY})
 	endif()
+	if (USE_POSTGRESQL)
+		target_link_libraries(${PROJECT_NAME}server ${PostgreSQL_LIBRARIES})
+	endif()
```

Comment:
Pardon me for being blind, but I thought I added it where required similarly to to the other database backends. I have also added to the repo in the case you feeling like adding something directly. Please let me know what else I can do. Thanks